src/main/java/com/zotov/edu/passportofficerestservice/service/PersonService.java

```diff
@@ -60,7 +60,7 @@ public void deletePersonById(String personId) {
     }
 
     public void checkIfPersonExists(String personId) {
-        if (!personsRepository.existsById(personId)) {
+        if (personsRepository.existsById(personId).equals(Boolean.FALSE)) {
```

Comment:
Let's move back to `boolean` because `Boolean` is this context is super non-obvious and error-prone

Reply:
replaced by boolean